tcms/management/migrations/0005_merge_conflicting_0004s.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.1.5 on 2019-01-24 16:50
```

Comment:
This migration needs to be removed and yours should follow after 0004_remove_sortkey_description